index.html

```diff
@@ -58,7 +59,7 @@ <h2>Software Developer</h2>
             <li><a href="#">Terminal</a></li>
         </ul>
     </section>
```

Comment:
- [ ] For this entire section, make it match the given design.
Your page
![Screenshot from 2021-09-17 02-27-25](https://user-images.githubusercontent.com/65501535/133701310-0f1ac20a-6e0a-433e-b892-6dff51292893.png)

Original page

![Screenshot from 2021-09-17 02-29-16](https://user-images.githubusercontent.com/65501535/133701361-039c4dba-ba0e-41df-bb4c-14c29cb4bd8d.png)